routes/orders.js

```diff
@@ -3,20 +3,152 @@ import express from 'express'
 
 const router = express.Router()
 
-router.get('/', async (req, res) => {
-  let orders
-  try {
-    orders = await models.orders.findAll()
-  } catch (err) {
-    console.log(err)
-    res.sendStatus(500)
-    return
-  }
-  if (orders) {
-    res.send(orders)
+/**
+ * @api {get} /action/:actionId Request action information
+ * @apiName GetAction
+ * @apiGroup Action
+ *
+ * @apiParam {Number} actionId Action's unique ID
+ *
+ * @apiSuccess {String} address The address of the action
+ * @apiSuccess {String} name The name of the action
+ * @apiSuccess {String} description A description of the action
+ * @apiSuccess {Number} duration Duration, in seconds, of the action
+ * @apiSuccess {Number} assetId The asset that this action is associated with, if any
+ * @apiSuccess {Number} redValue If this is a background color changing action, the value of red that should be used. Should be between 0-255.
```

Comment:
This should not be here, copy/paste leftovers.

Reply:
I already updated the routes. Can you see the new changes?